### PR TITLE
Incorporate new Telemetry package

### PR DESCRIPTION
**Pull request by rynowak**
/cc @Eilon

### CODE REVIEW

**Comment by rynowak** [comment]
@Eilon  ping

File: src/Microsoft.Framework.TelemetryAdapter/project.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
+  "version": "1.0.0-*",
+  "description": "An event notification system.",
```
Comment (pranavkm) [inline src/Microsoft.Framework.TelemetryAdapter/project.json]: > A telemetry system?

**Comment by pranavkm** [comment]
:shipit:

File: src/Microsoft.Framework.TelemetryAdapter/project.json (hunk 1 above)
Comment (rynowak) [inline src/Microsoft.Framework.TelemetryAdapter/project.json]: Yeah this needs to be updated

**Pull request closed without merging**